### PR TITLE
Prep for Release 0.25.8

### PR DESCRIPTION
**Pull request by bigfishdesign13**
## This PR does the following:

- prep for release `v0.25.8`

### CODE REVIEW

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr831-ua1o5lsykadl3xrwuquza24opl01pjql.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/61d705b55f46ed13d8d611cd

File: src/components/Select/Select.stories.mdx [hunk 1]
```diff
@@ -39,7 +39,7 @@ import DSProvider from "../../theme/provider";
 | Component Version | DS Version |
 | ----------------- | ---------- |
 | Added             | `0.7.0`    |
-| Latest            | `0.25.1`   |
+| Latest            | `0.25.8`   |
```
Comment (EdwinGuzman) [inline src/components/Select/Select.stories.mdx:42]: I think this should be `0.25.6`.

Comment (bigfishdesign13) [reply src/components/Select/Select.stories.mdx:42]: What about `DSD-667`?

**Review by EdwinGuzman — APPROVED** [review]

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr831-ua1o5lsykadl3xrwuquza24opl01pjql.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/61d705b55f46ed13d8d611cd

**Pull request merged**